yt/yt/client/formats/config.h

```diff
@@ -415,4 +415,24 @@ DEFINE_REFCOUNTED_TYPE(TSkiffFormatConfig)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+class TYamlFormatConfig
+    : public NYTree::TYsonStruct
+{
+public:
+    //! Write explicit tag "!yt/uint" for uint64 data type.
+    //! Use this option if you want to preserve information about
+    //! the original YT type (without it, numbers in range [0, 2^63-1]
+    //! will always be written as integers).
+    //! Option has no effect for parsing.
+    bool WriteUintTag;
```

Comment:
Why this option has no effect for parsing?

Reply:
I feel like it is weird to have an option that makes our parser discard !!yt/uint tag, what can be use of it? If somebody puts this tag, they probable mean it and I feel like trusting it unconditionally is the best course of actions.